module/Application/src/Factory/AccountControllerFactory.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: benmorris
+ * Date: 29/11/2016
+ * Time: 14:02
+ */
+
+namespace Application\Factory;
+
+class AccountControllerFactory
+{
+    public function __invoke($sm) {
+        $user = $sm->get(\Application\Model\UserModel::class);
```

Comment:
change to user moodel var name and swap fully qualified namespaces for 'use' statements